modules/tinymce/src/core/test/ts/browser/DragnDropCEFTest.ts

```diff
@@ -0,0 +1,82 @@
+import { Mouse, UiFinder } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { SugarLocation } from '@ephox/sugar';
+import { TinyAssertions, TinyDom, TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.DragnDropCEFTest', () => {
+  const getBaseCEFElement = (name: string) => [ `<div class="${name}" style="margin: 40px;" contenteditable="false">`,
```

Comment:
Does the HTML need to be this complex to simulate the fix here I think a div with a width and height should be enough.

Reply:
ok, I simplified it